velox/docs/develop/window.rst

```diff
@@ -0,0 +1,202 @@
+========================
+Window Operator
+========================
+
+Velox supports window function evaluation using Window operator. In this guide
```

Comment:
> In this guide we will discuss some intricate design issues in this operator.

'issues' sounds like 'problems'. Is this intentional? 

Reading ahead it seems that this article focuses specifically on window frames. Would it make sense to state that clearly and rename the file and the doc title to reflect that?

Do you expect readers to be familiar with window functions, e.g. the content available in https://facebookincubator.github.io/velox/functions/presto/window.html ? Is this an extension of that documentation? Maybe make this clear.

BTW, where is this document referenced from?

Reply:
Right now, I have material on window frames since there have many details that needed a doc explanation.

I'm in 2 minds if I should write an explanation of the window function API for a developer adding a new function. In the past, we felt that it was explanatory from the code. So I named this window.rst so that it can be extended in the future if required.

Added a reference to the doc from the top level index.